### PR TITLE
Update Zoom module to v1

### PR DESCRIPTION
**Pull request by igorschechtel**
Removed:
- `manager.js` and `manager.test.js` files
- Models folder

Added:
- `package.json` file
- `definition.js` file
- Tests (`api.test.js` and `auther.test.js`)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @friggframework/api-module-zoom@1.0.1-canary.8.a2fb172.0
  # or 
  yarn add @friggframework/api-module-zoom@1.0.1-canary.8.a2fb172.0
  ```
</details>

### CODE REVIEW

File: packages/zoom/api.js [hunk 1]
```diff
@@ -8,7 +7,7 @@ class Api extends OAuth2Requester {
         this.baseUrl = `https://api.zoom.us/v2/`;
         this.client_id = process.env.ZOOM_CLIENT_ID;
         this.client_secret = process.env.ZOOM_CLIENT_SECRET;
-        this.redirect_uri = 'https://29a3f7035dbb.ngrok.io';
+        this.redirect_uri = process.env.REDIRECT_URI;
```
Comment (leofmds) [inline packages/zoom/api.js:10]: You can remove this line. The parent class will handle the `redirect_uri` attribution. The user needs to pass the variable to work properly, but it is best than assigning the env to the redirect every time

**Review by leofmds — CHANGES_REQUESTED** [review]
Looks good, just a minor change

**Review by leofmds — APPROVED** [review]
All good 👍🏻

**Comment by MichaelRyanWebber** [comment]
This has been merged as part of https://github.com/friggframework/api-module-library/pull/10

**Pull request closed without merging**